lib/potluck.rb

```diff
@@ -0,0 +1,26 @@
+class Potluck 
+  attr_reader :date, 
+              :new_dish,
+              :dishes
+            
+
+  def initialize(date)
+    @date = date 
+    @dishes = []
+    @new_dish = new_dish
+  end 
+  
+  def dishes 
+    @dishes 
+  end 
+  
+  def add_dish(new_dish)
+    new_dish
+  end 
+  
+  def length
```

Comment:
Make sure you pay attention to the spec!  We are looking to call `potluck.dishes.length`, not `potluck.length`.  also, I think line 22 probably belongs in the `add_dish` method.